scripts/qiita

```diff
@@ -384,13 +386,20 @@ def unlock():
               help='The amount of time to show message, in seconds')
 @click.option('--message', required=True, type=str,
               help="Message to show users")
-def sysmessage(message, time):
-    r_client.setex('sysmessage', message, time)
+@click.option('--banner', is_flag=True,
+              help="If set, show as site banner instead of a standard message")
+def sysmessage(message, time, banner):
```

Comment:
I'm gonna leave this to other devs, but the `--time` parameter makes sense for the redis system. However, for the messaging system that we are putting in place, I think the most natural input will be to provide hour, minutes, day, month and year. Either in a specified format or in multiple parameters.

I'm bringing this up because I saw today that when we create a system message we end up putting a really big number so we can clear it up afterwards... this makes the functionality effectively useless as we are not going to count the seconds that we need until the day that we are gonna do the upgrade...